source/fuzz/transformation.cpp

```diff
@@ -357,6 +358,9 @@ std::unique_ptr<Transformation> Transformation::FromMessage(
     case protobufs::Transformation::TransformationCase::kSwapCommutableOperands:
       return MakeUnique<TransformationSwapCommutableOperands>(
           message.swap_commutable_operands());
+    case protobufs::Transformation::TransformationCase::kSwapFunctionVariables:
+      return MakeUnique<TransformationSwapFunctionVariables>(
+          message.swap_function_variables());
```

Comment:
Better to keep these sorted by the name of the enumeration. In your case, just swap it with the next `case` block.

Reply:
done